mava/adders/reverb/base.py

```diff
@@ -54,6 +40,9 @@ class Step(NamedTuple):
     extras: Dict[str, types.NestedArray]
 
 
+Trajectory = Step
```

Comment:
Is there a reason we do this instead of calling the above class `Trajectory`?

Reply:
I also found this a bit weird. I think acme is planning on possibly having the trajectory class change in the future, but for now it is the same as the `Step` class. I think we should leave it for now incase the Trajectory class in the near future. Thoughts?

Reply:
Ok makes sense, not sure if this was from our side or theirs. But happy to keep it as is for now.